cli/scan-bitbucket-repo-for-eol-php.js

```diff
@@ -0,0 +1,31 @@
+const dotenv = require('dotenv')
+const { getPhpVersionVulnerabilitiesOfBitbucketRepo } = require('../index')
+
+dotenv.config()
+
+const username = process.env.BITBUCKET_USERNAME
+const appPassword = process.env.BITBUCKET_APP_PASSWORD
+
+const repo = process.argv[2]
+const versionsCsvUrl = process.argv[3]
+
+if (!username) {
+  console.error('Please provide a BITBUCKET_USERNAME environment variable')
+  process.exitCode = 1;
+} else if (!appPassword) {
+  console.error('Please provide a BITBUCKET_APP_PASSWORD environment variable')
+  process.exitCode = 1;
+} else if (!repo) {
+  console.error('Please specify a Bitbucket repo in the form "workspace/repo-slug"')
```

Comment:
Your README is excellent in that it provides usage, but do you also want to print out the correct usage here? Something like:
```
Usage:

node cli/scan-bitbucket-repo-for-eol-php.js REPO VERSIONS_URL
```
Actually, I went back to look in the README for the usage of this script and I didn't see it. Is that because this is only for internal use?

Reply:
Hmm, good thought. I probably hadn't added this to the README because it was a late decision to include this file. Originally, it was just for me to test this new feature as I developed it, but it ended up being so useful I decided to keep it.

Reply:
For now, I'm going to proceed with the CLI error message as-is. Big picture, we need a better CLI anyway, perhaps as an external library that uses this one.